package/MDAnalysis/lib/distances.py

```diff
@@ -1102,9 +1102,9 @@ def _nsgrid_capped_self(reference, max_cutoff, min_cutoff=None, box=None,
             gridsearch = FastNS(max_cutoff, reference, box=box)
             results = gridsearch.self_search()
 
-        pairs = results.get_pairs()[::2, :]
```

Comment:
Here we used to double save pairs then slice them, so instead you can just not double save

Reply:
Does anyone depend on that behavior? This is the unit test change below?